docs/content/v0.20/packaging.md

```diff
@@ -47,19 +51,40 @@ The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-ge
 
 ### Central container image
 
-Currently at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`.
+KubeStellar has one container image, for what is called the
+KubeStellar operator. This operator installs the needed custom
+resource _definition_ objects if they are not already present, and is
+a controller-manager hosting the relevant central controllers.
 
-`make ko-build-push` will create that image. `make ko-build-local` will make a local image for just the local platform.
+The image repository is
+`ghcr.io/kubestellar/kubestellar/kubestellar-operator`. There is
+currently just one supported tag there, `0.20.0-alpha.1`. That was
+built from the git repo contents with the tag `v0.20.0-alpha.1`.
+
+`make ko-build-push` will build and push that image, tagging it with
+the value of the `make` variable `IMAGE_TAG` (which defaults to
+`0.20.0-alpha.1`).
+
+`make ko-build-local` will make a local image for just the local
+platform.
```

Comment:
Maybe add that this is used for local testing ?

Reply:
Done.